modules/core/client/directives/page-title.client.directive.js

```diff
@@ -19,13 +19,17 @@
 
       function listener(event, toState) {
         var applicationCoreTitle = 'MEAN.js',
-          separeteBy = ' - ';
+          separeteBy = ' - ',
+          stateTitle = applicationCoreTitle + separeteBy;
+
+        _.forEach(toState.name.split('.'), function(value, index) {
+          stateTitle = stateTitle + _.capitalize(value) + separeteBy;
```

Comment:
clearly, i suggest the addition. it's an example in order to discuss about this subject.